roc/convert.go

```diff
@@ -4,6 +4,8 @@ import (
 	"C"
 )
 
+type char = C.char
```

Comment:
nit: let's use `char` instead of `C.char` below

Reply:
done